.github/workflows/cli-tests.yaml

```diff
@@ -85,15 +111,10 @@ jobs:
           key: go-${{ runner.os }}-${{ hashFiles('go.sum') }}
 
       # Use main devbox for now to ensure it supports runx
-      - run:  go run ./cmd/devbox run fmt
-      
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3.7.0
-        with:
-          args: "--out-${NO_FUTURE}format colored-line-number --timeout=10m"
-          skip-cache: true
+      - run:  go run ./cmd/devbox run lint
```

Comment:
Use the built artifact here too?

Reply:
I changed this, but I think longer term we can use the released devbox instead for speed since this is one of the first tests we want to see complete.